### DISCUSSION TITLE
Private repo to public repo release

### DISCUSSION THREAD

**Discussion by developerautomationkit** [cawa-93/vite-electron-builder#1051] [Q&A]
Seeing that you are hard at work again . I wanted to ask  if you know by chance how to configure electron builder + github actions for pushing to a Private repo to public repo release in github actions ?

Iam using older version of you template and just wondered if you have any idea by chance what could be wrong

this is a issue i created couple days ago : Just cant figure out why electron builder does not seem to work with Custom Personal Accces token ? Any thoughts on how you would aproach this release on Private repo to public repo release ? Greetings

https://github.com/electron-userland/electron-builder/issues/8676

**Comment by cawa-93** [comment]
https://github.com/cawa-93/vite-electron-builder/blob/999607ba5a199c5ed4fb1ad99e2e80a259818003/.github/workflows/deploy.yml#L40-L42

**Comment by developerautomationkit** [comment]
> https://github.com/cawa-93/vite-electron-builder/blob/999607ba5a199c5ed4fb1ad99e2e80a259818003/.github/workflows/deploy.yml#L40-L42

to my knowledge the github.token is restricted to the current repo ? 
Is that not the case ? any experience regarding having to use Personal acces token PAT ?

I feel like this is a very common usecase having a private repo and releasing to a public one . But for the love of god cant figure out why  the github actions doesnt work with electron builder  PAT :D

But I guess your github create aproach may work with pat .. gotta try that

**Comment by cawa-93** [comment]
Currently I don't use electron builder for publishing. I use gh cli instead to have for control. And I believe you can authorise it and create release in one repo from another. Just ensure you provide correct credentials with enough permission in credentials and workflow
